manager/orchestrator/errors.go

```diff
@@ -0,0 +1,16 @@
+package orchestrator
+
+import (
+	"errors"
+)
+
+var (
+	errReplicatedOrchestratorStarted    = errors.New("ReplicatedOrchestrator: already started")
+	errReplicatedOrchestratorNotStarted = errors.New("ReplicatedOrchestrator: not started")
+
+	errGlobalOrchestratorStarted    = errors.New("GlobalOrchestrator: already started")
+	errGlobalOrchestratorNotStarted = errors.New("GlobalOrchestrator: not started")
+
+	errTaskReaperStarted    = errors.New("TaskReaper: already started")
+	errTaskReaperNotStarted = errors.New("TaskReaper: not started")
+)
```

Comment:
I prefer not to move the errors to a separate file - especially since all of these errors are specific to the code in a particular file.